docs/operations-guide/migrating-from-h2.md

```diff
@@ -1,6 +1,13 @@
 # Migrating from using the H2 database to MySQL or Postgres
 
-If you decide to use the default application database (H2) when you initially start using Metabase, but later decide that you'd like to switch to a more production-ready database such as MySQL or Postgres, we make the transition easy for you.
+If you decide to use the default application database (H2) when you initially start using Metabase, but later decide that you'd like to switch to a more production-ready database such as MySQL or Postgres, you're in the right place.
+
+## Before you migrate
+
+- Avoid upgrading and migrating at the same time, since it can cause problems with one of database schemas not matching.
+- You must be able to connect to the target MySQL or Postgres database in whatever environment you're running this migration command in. So, if you are attempting to move the data to a cloud database, make sure you can connect to that database.
```

Comment:
Q: do we only support MySQL and Postgres as replacements for H2? This makes it sound as if that's so.

Reply:
Yes, only Postgres and MySQL/MariaDB are supported as appdb - besides H2.